examples/demo/site/scripts/node.sql

```diff
@@ -11,7 +11,8 @@ CREATE TABLE "nodes"(
   "author" INTEGER,
   "publish" DATETIME,
   "created" DATETIME NOT NULL,
-  "changed" DATETIME NOT NULL
+  "changed" DATETIME NOT NULL,
+  "deleted_at" DATETIME NULL
```

Comment:
I think we should adopt a different design here.
And go for adding a "status"
and then, "deleted" would be   status = deleted_id  , and deleted_at .. would be "changed" date.